data/sql/updates/pending_db_world/rev_1607274868483442300.sql

```diff
@@ -0,0 +1,326 @@
+(84535, 84635, 0, 0, 3, 0, 0),
+(84535, 84639, 0, 0, 3, 0, 0),
+(84535, 84618, 0, 0, 3, 0, 0),
+(84535, 84619, 0, 0, 3, 0, 0),
+(84535, 84623, 0, 0, 3, 0, 0),
+(84535, 84624, 0, 0, 3, 0, 0),
+(84535, 84625, 0, 0, 3, 0, 0),
+(84535, 84628, 0, 0, 3, 0, 0),
+(84535, 84617, 0, 0, 3, 0, 0),
+(84535, 84634, 0, 0, 3, 0, 0);
+*/
+
+-- Pack right after ramps in second level of Firemaws patrol area
+DELETE FROM `creature_formations` WHERE `memberGUID` IN (84536, 61998, 84523, 84522, 84650, 84691, 84648, 84651, 84652, 84688, 84689, 84653, 84654, 84655);
+/*
+-- This is a blizzlike fix, although unsupported/buggy core code on the sunwell base
+INSERT INTO `creature_formations` (`leaderGUID`, `memberGUID`, `dist`, `angle`, `groupAI`, `point_1`, `point_2`) VALUES
+(84536, 84536, 0, 0, 3, 0, 0),
+(84536, 61998, 0, 0, 3, 0, 0),
+(84536, 84523, 0, 0, 3, 0, 0),
+(84536, 84522, 0, 0, 3, 0, 0),
+(84536, 84650, 0, 0, 3, 0, 0),
+(84536, 84691, 0, 0, 3, 0, 0),
+(84536, 84648, 0, 0, 3, 0, 0),
+(84536, 84651, 0, 0, 3, 0, 0),
+(84536, 84652, 0, 0, 3, 0, 0),
+(84536, 84688, 0, 0, 3, 0, 0),
+(84536, 84689, 0, 0, 3, 0, 0),
+(84536, 84653, 0, 0, 3, 0, 0),
+(84536, 84654, 0, 0, 3, 0, 0),
+(84536, 84655, 0, 0, 3, 0, 0);
+*/
+-- Pack to the left wall in second level of Firemaws patrol area
+DELETE FROM `creature_formations` WHERE `memberGUID` IN (84537, 84524, 61999, 67032, 84525, 84759, 84692, 84761, 85601, 84763, 85581, 84760, 84762, 84840);
+/*
+-- This is a blizzlike fix, although unsupported/buggy core code on the sunwell base
+INSERT INTO `creature_formations` (`leaderGUID`, `memberGUID`, `dist`, `angle`, `groupAI`, `point_1`, `point_2`) VALUES
+(84537, 84537, 0, 0, 3, 0, 0),
+(84537, 84524, 0, 0, 3, 0, 0),
+(84537, 61999, 0, 0, 3, 0, 0),
+(84537, 67032, 0, 0, 3, 0, 0),
+(84537, 84525, 0, 0, 3, 0, 0),
+(84537, 84759, 0, 0, 3, 0, 0),
+(84537, 84692, 0, 0, 3, 0, 0),
+(84537, 84761, 0, 0, 3, 0, 0),
+(84537, 85601, 0, 0, 3, 0, 0),
+(84537, 84763, 0, 0, 3, 0, 0),
+(84537, 85581, 0, 0, 3, 0, 0),
+(84537, 84760, 0, 0, 3, 0, 0),
+(84537, 84762, 0, 0, 3, 0, 0),
+(84537, 84840, 0, 0, 3, 0, 0);
+*/
+
+-- Pack to the right wall in second level of Firemaws patrol area
+DELETE FROM `creature_formations` WHERE `memberGUID` IN (84538, 84527, 84389, 84390, 84526, 85611, 85616, 85622, 85612, 85609, 85620, 85613, 85621, 85623);
+/*
+-- This is a blizzlike fix, although unsupported/buggy core code on the sunwell base
+INSERT INTO `creature_formations` (`leaderGUID`, `memberGUID`, `dist`, `angle`, `groupAI`, `point_1`, `point_2`) VALUES
+(84538, 84538, 0, 0, 3, 0, 0),
+(84538, 84527, 0, 0, 3, 0, 0),
+(84538, 84389, 0, 0, 3, 0, 0),
+(84538, 84390, 0, 0, 3, 0, 0),
+(84538, 84526, 0, 0, 3, 0, 0),
+(84538, 85611, 0, 0, 3, 0, 0),
+(84538, 85616, 0, 0, 3, 0, 0),
+(84538, 85622, 0, 0, 3, 0, 0),
+(84538, 85612, 0, 0, 3, 0, 0),
+(84538, 85609, 0, 0, 3, 0, 0),
+(84538, 85620, 0, 0, 3, 0, 0),
+(84538, 85613, 0, 0, 3, 0, 0),
+(84538, 85621, 0, 0, 3, 0, 0),
+(84538, 85623, 0, 0, 3, 0, 0);
+*/
```

Comment:
shall we remove the commented code?

Reply:
Removed, will open a new issue for it later, because it's unrelated to this PR/initial issue. It's global.